dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/LinkageCheckerMain.java

```diff
@@ -73,6 +81,10 @@ public static void main(String[] arguments)
     }
 
     System.out.println(SymbolProblem.formatSymbolProblems(symbolProblems));
-  }
 
+    if (!artifactProblems.isEmpty()) {
+      System.out.println("\n");
+      System.out.println(ArtifactProblem.formatProblems(artifactProblems));
```

Comment:
This is the goal of this PR. LinkageCheckerMain prints artifact problems if any.